### PR TITLE
Fix mockRDSClient.DescribeDBSnapshots list make

### PR DESCRIPTION
**Pull request by ifosch**
I always get confused about how Go's `append` makes space in lists.

### CODE REVIEW

File: mcc/odin/odin/mock_rds_client_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (m mockRDSClient) DescribeDBSnapshots(
 	if describeParams.DBInstanceIdentifier != nil {
 		snapshots = m.dbSnapshots[*describeParams.DBInstanceIdentifier]
 	} else {
-		snapshots = make([]*rds.DBSnapshot, len(m.dbSnapshots))
```
Comment (ifosch) [inline mcc/odin/odin/mock_rds_client_test.go:29]: This was causing the list to have one empty (`nil`) item in the final list.

Comment (ifosch) [reply mcc/odin/odin/mock_rds_client_test.go:29]: Well, actually, as many `nil` items as `len(m.dbSnapshots)` value.

**Comment by nicolas-fricke** [comment]
+1 I think

File: mcc/odin/odin/mock_rds_client_test.go (hunk 1 above)
Comment (miguelbernadi) [reply mcc/odin/odin/mock_rds_client_test.go:29]: better not to specify a size and let runtime allocate space as needed.

**Comment by miguelbernadi** [comment]
+1

**Pull request merged**